### PR TITLE
Update all configs simplified

### PR DESCRIPTION
**Pull request by ida3m0n**
This will impact directly into PPA & Cloud Marketplaces.

Cleaner configs , minimal.
Removed bash lines from rskj.postinst. Reason: No longer used.
Removed ubuntu artful directory, since its no longer supported.

### CODE REVIEW

**Comment by ida3m0n** [comment]
> Please rebase to fix conflicting files

Done !

File: rskj-ubuntu-installer/config/regtest.conf [hunk 1]
```diff
@@ -231,36 +76,8 @@ wallet {
...
-    addresses {
-        # initial punishment duration (in minutes, default = 10 minutes)
-        duration: 12
-        # punishment duration increment (in percentage, default = 10)
-        increment: 10
-        # maximum punishment duration (in minutes, default = 1 week)
-        maximum: 6000
-    }
-}
+solc.path = <SOLC_PATH>
```
Comment (tinchou) [inline rskj-ubuntu-installer/config/regtest.conf]: The idea is that only `regtest` has `solc.path` enabled, right? Great idea.

File: rskj-ubuntu-installer/config/testnet.conf [hunk 2]
```diff
@@ -1,161 +1,8 @@
...
-
-    # maximum blocks hashes to ask sending GET_BLOCK_HASHES msg we specify number of block we want to get, recomended value [1..1000]
-    # Default: unlimited
-    max.hashes.ask = 10000
-
-    # minimal peers count used in sync process 
-    # sync may use more peers than this value but always trying to get at least this number from discovery
-    peer.count = 10
-}
+prune.enabled = true
```
Comment (tinchou) [inline rskj-ubuntu-installer/config/testnet.conf]: Testnet is now in Orchid, so `prune` is not necessary anymore :D

File: rskj-ubuntu-installer/config/regtest.conf [hunk 3]
```diff
@@ -273,28 +90,5 @@ miner {
 
     # this is a secret passphrase that is used to derive the address where the miner gets the reward.
     # please note this is stored in a local wallet and not recommended for production.
-    coinbase.secret = <COINBASE_SECRET>
```
Comment (tinchou) [inline rskj-ubuntu-installer/config/regtest.conf]: miner server is enabled by default in regtest. you could simplify to `miner.coinbase.secret = <COINBASE_SECRET>`

File: rskj-ubuntu-installer/config/regtest.conf (hunk 1 above)
Comment (ida3m0n) [reply rskj-ubuntu-installer/config/regtest.conf]: Yes.

**Pull request merged**